### PR TITLE
fix(kit): update statically `HostBinding` in `line-clamp`

### LINKED ISSUE #3087: 🐞 - Expression has changed after it was checked when using LineClamp
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/angular-s61afv?file=src%2Fapp%2Fapp.module.ts,src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.less&hideExplorer=1

### Is this issue blocking you?

Non-Blocking

### Description

Hi! When I use tui-line-clamp element with *ngIf directive, I got error `ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value for 'max-height': 'null'. Current value: '20'`

Example:
``` ts
// Component
ngOnInit() {
  setTimeout(() => {
    this.value = 'fake-value';
  }, 2000);
}
```

``` html
<!-- Template -->
<div *ngIf="value">
  <tui-line-clamp [content]="value" [lineHeight]="20" [linesLimit]="1"></tui-line-clamp>
</div>
```

Снимок экрана 2022-11-18 в 20 12 03

### Angular version

14.2.11

### Taiga UI version

3.10.0

### Which browsers have you used?

- [X] Chrome
- [X] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3087

image

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/line-clamp/examples/1/index.ts [hunk 1]
```diff
@@ -9,4 +10,17 @@ import {encapsulation} from '@demo/emulate/encapsulation';
...
     encapsulation,
 })
-export class TuiLineClampExample1 {}
+export class TuiLineClampExample1 {
+    value?: string;
+
+    constructor(@Inject(TUI_IS_CYPRESS) private readonly isCypress: boolean) {}
+
+    ngOnInit(): void {
+        setTimeout(
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/line-clamp/examples/1/index.ts:19]: let's rewrite it in rxjs way

```ts
export class TuiLineClampExample1 {
    value$ = timer(this.isCypress ? 0 : 4000).pipe(
        map(() => `${'async fake value, '.repeat(10)}end!`)
    );

    constructor(@Inject(TUI_IS_CYPRESS) private readonly isCypress: boolean) {}
}
```

```html
<div
    *ngIf="value$ | async as value; else loading"
>
```

Comment (nsbarsukov) [reply projects/demo/src/modules/components/line-clamp/examples/1/index.ts:19]: nit

File: projects/kit/components/line-clamp/line-clamp.template.html [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <div
-    *tuiLet="lineClamp$ | async as lineClamp"
     tuiHintCustomPosition
     class="t-wrapper"
     [tuiHint]="computedContent"
-    [style.-webkit-line-clamp]="lineClamp"
-    [style.word-break]="(lineClamp || 0) > 1 ? 'break-word' : 'break-all'"
+    [style.-webkit-line-clamp]="lineClamp$ | async"
+    [style.word-break]="((lineClamp$ | async) || 0) > 1 ? 'break-word' : 'break-all'"
```
Comment (nsbarsukov) [inline projects/kit/components/line-clamp/line-clamp.template.html:6]: two subscriptions of the same observable :(

Can we avoid it somehow ?

Comment (nsbarsukov) [reply projects/kit/components/line-clamp/line-clamp.template.html:6]: Why `*tuiLet` is not appropriate solution ?

File: projects/demo/src/modules/components/line-clamp/examples/1/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/line-clamp/examples/1/index.ts]: fixed

File: projects/kit/components/line-clamp/line-clamp.template.html (hunk 2 above)
Comment (splincode) [reply projects/kit/components/line-clamp/line-clamp.template.html]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Comment by splincode** [comment]
@vladimirpotekhin hello, please review me)

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**